shared/src/validation/validation-schemas.ts

```diff
@@ -1,3 +1,3 @@
 export { EamWorkerCreate } from './eam-worker/worker';
 export { eamMasterSignUp, eamMasterSignIn } from './eam-master/eam-master';
-export { eamGroupCreate } from './eam-group/eam-group';
+export { eamGroupCreate, eamGroupConfigurate } from './eam-group/eam-group';
```

Comment:
```suggestion
export { eamGroupCreate, eamGroupConfigurate } from './eam-group/eam-group';
```
what are the differences between eamGroupCreate and eamGroupConfigurate?